sky/skylet/constants.py

```diff
@@ -316,6 +316,15 @@
 # API server.
 SKY_API_SERVER_URL_ENV_VAR = f'{SKYPILOT_ENV_VAR_PREFIX}API_SERVER_ENDPOINT'
 
+# The name of the environment variable that specified the max parallelism of
+# API server for long request. An best-effort estimation is used if not set.
+API_SERVER_LONG_REQ_PARALLELISM_ENV_VAR = (
+    f'{SKYPILOT_ENV_VAR_PREFIX}MAX_PARALLEL_LONG_WORKERS')
+# The name of the environment variable that specified the max parallelism of
+# API server for short request. An best-effort estimation is used if not set.
+API_SERVER_SHORT_REQ_PARALLELISM_ENV_VAR = (
+    f'{SKYPILOT_ENV_VAR_PREFIX}MAX_PARALLEL_SHORT_WORKERS')
```

Comment:
After thinking more, how about we just remove these env vars support, because no user would understand how to set these numbers. Let's just avoid them for simplicity.

Reply:
My previous intent is to provide a knob for users who encounter resource issues without patching the code. Setting `SKYPILOT_POD_CPU_CORE_LIMIT` and `SKYPILOT_POD_MEMORY_GB_LIMIT` can also do the trick with some indirections though. I'm okay with simplifying the code now and add it back only when someone really needs it.